---
layout: default
---
# GB0258

## Graph-Break Type
*Short name describing what triggered the graph break*

collections.deque() with bad arguments

## Context
*Values or code snippet captured at the break point*

args={args}, kwargs={kwargs}

## Explanation
*Explanation of why the graph break was triggered*

Detected call to collections.deque() with bad arguments.

## Hints
*Hints on how to resolve the graph break*

- Fix the call to collections.deque().
- Dynamo has detected that tracing the code will result in an error when running in eager. Please double check that your code doesn't contain a similar error when actually running eager/uncompiled.


## Additional Information

<!-- ADDITIONAL INFORMATION START - Add custom information below this line -->

<!-- ADDITIONAL INFORMATION END -->


[Click here to add Additional Info](https://github.com/pytorch-labs/compile-graph-break-site/edit/main/docs/gb/gb0258.md)

[Back to Registry](../index.html)
